FakeTrello/Models/UserTag.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class UserTag
+    {
```

Comment:
Try exploring using a navigation property on your card class for this instead of a class. You could have an owner and a tags property.